src/util/ori.js

```diff
@@ -8,6 +8,12 @@ const postalMappingExceptions = {
   GM: 'GU',
 }
 
+export const reshapeData = data =>
```

Comment:
It would be good to add some tests for this function so that we can use that as documentation of how this function is used.